File: app/src/main/java/tpcreative/co/qrscanner/helper/TimeHelper.java
package tpcreative.co.qrscanner.helper;
import java.text.ParseException;
import java.text.SimpleDateFormat;
import java.util.Date;
import java.util.Locale;

public class TimeHelper {

    private  static TimeHelper mInstance;
    private TimeHelper(){}
    public static TimeHelper getInstance(){
        if (mInstance==null){
            mInstance = new TimeHelper();
        }
        return mInstance;
    }

    public Date getDateTime(){
        return new Date();
    }
}

File: app/src/main/java/tpcreative/co/qrscanner/helper/TimeHelper.kt
package tpcreative.co.qrscanner.helper

import java.util.*

class TimeHelper private constructor() {
    fun getDateTime(): Date? {
        return Date()
    }

    companion object {
        private var mInstance: TimeHelper? = null
        fun getInstance(): TimeHelper? {
            if (mInstance == null) {
                mInstance = TimeHelper()
            }
            return mInstance
        }
    }
}